platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -174,3 +174,67 @@ include::sample[dir="snippets/bestPractices/avoidInternal/groovy",files="build.g
 === Tags
 
 `<<tags_reference.adoc#tag:upgrades,#upgrades>>`
+
+[[favor_composite_builds]]
+== Favor Composite Builds over `buildSrc`
+:keywords: structuring-builds,composite-builds,buildSrc
+
+You should setup a <<composite_builds.adoc#defining_composite_builds,Composite Build>> (often called an "included build") to hold your build logic—including any custom plugins, convention plugins, and other build-specific customizations.
+
+=== Explanation
+
+The preferred location for build logic is an included build named `build-logic`, **not** in `buildSrc`.
```

Comment:
I would change the wording about naming the included build. Right now it suggests that it **has to** be named as such.